src/main/telemetry/frsky.h

```diff
@@ -22,10 +22,13 @@ typedef enum {
     FRSKY_VFAS_PRECISION_HIGH
 } frskyVFasPrecision_e;
 
+typedef void frSkyTelemetryInitFrameFunc(void);
```

Comment:
Although not all code complies, standard suffix for function pointer is `FnPtr` so this should be `frSkyTelemetryInitFrameFnPtr`